analyzer/codechecker_analyzer/pre_analysis_manager.py

```diff
@@ -129,7 +129,6 @@ def pre_analyze(params):
     except Exception as ex:
         LOG.debug_analyzer(str(ex))
         traceback.print_exc(file=sys.stdout)
-        raise
```

Comment:
Also, I strongly disagree with this blanket removal, even if we were to go through with what this patch proposes -- this gets rid of all exceptions, not just FileNotFoundError.